src/ucs/sys/math.h

```diff
@@ -65,26 +65,29 @@ BEGIN_C_DECLS
 #define ucs_align_up_pow2_ptr(_ptr, _alignment) \
     ((typeof(_ptr))ucs_align_up_pow2((uintptr_t)(_ptr), (_alignment)))
 
-#define ucs_roundup_pow2(n) \
+#define ucs_roundup_pow2(_n) \
     ({ \
-        typeof(n) pow2; \
-        ucs_assert((n) >= 1); \
-        for (pow2 = 1; pow2 < (n); pow2 <<= 1); \
+        typeof(_n) pow2; \
+        ucs_assert((_n) >= 1); \
+        for (pow2 = 1; pow2 < (_n); pow2 <<= 1); \
         pow2; \
     })
 
 /* Return values: 0 - aligned, non-0 - unaligned */
-#define ucs_check_if_align_pow2(n, p) ((n) & (p-1))
+#define ucs_check_if_align_pow2(_n, _p) ((_n) & (_p - 1))
 
 /* Return values: off-set from the alignment */
-#define ucs_padding_pow2(n, p) ucs_check_if_align_pow2(n, p)
+#define ucs_padding_pow2(_n, _p) ucs_check_if_align_pow2(_n, _p)
 
-#define UCS_MASK_SAFE(i) \
-    (((i) >= 64) ? ((uint64_t)(-1)) : UCS_MASK(i))
+#define UCS_MASK_SAFE(_i) \
+    (((_i) >= 64) ? ((uint64_t)(-1)) : UCS_MASK(_i))
 
 #define ucs_div_round_up(_n, _d) \
     (((_n) + (_d) - 1) / (_d))
 
+/* Return values: `0` - (_x == 0), `-1` - (_x < 0), `1` - (_x > 0) */
+#define ucs_signum(_x) ((_x > 0) - (_x < 0))
```

Comment:
( ) around _x

Reply:
it has been removed and will be added in the PR with hashing